### ISSUE TITLE
RA Auto-Start Setting not in the Wiki

### ISSUE THREAD

**Issue by JazzJohannes** [cbepx-me/Anbernic-H700-RG-xx-StockOS-Modification#76]
The retroarch setting for auto-resume short-pressing "A" or long-pressing "A" is not explained on the wiki yet.

**Comment by cbepx-me** [comment]
Sorry, I haven't updated the wiki recently because I've been busy.

**Comment by cbepx-me** [comment]
Let me give a brief explanation here. When you short press A to start the game, automatic saving and loading of the game will be enabled. When you long press A to start the game, automatic saving and loading of the game will be disabled.

**Comment by JazzJohannes** [comment]
> Let me give a brief explanation here. When you short press A to start the game, automatic saving and loading of the game will be enabled. When you long press A to start the game, automatic saving and loading of the game will be disabled.

Yeah, and that just needs adding to the Wiki then, perfect! Thank you!

**Comment by cbepx-me** [comment]
At present, most of the settings and functions have been explained in the apps, and the wiki will not be updated temporarily.
